pkg/http/controller.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func (s *HTTPServer) CreateNetwork(c *gin.Context) {
+	// services.S3Service().UploadContent("file_5", " content 5 ")
```

Comment:
Our all core logic will be developed in api package. I think you can add s3 as member of TitondAPI.